cmake/FetchHolohubOperator.cmake

```diff
@@ -71,6 +88,7 @@ function(fetch_holohub_operator OPERATOR_NAME)
             cmake/pydoc/
             operators/operator_util.hpp
         && git checkout ${ARGS_BRANCH}
+        ${PATCH_COMMAND}
```

Comment:
Why not use CMake's [PATCH_COMMAND](https://cmake.org/cmake/help/latest/module/ExternalProject.html#patch-step-options)? Looks like this is currently adding patching as part of the DOWNLOAD_COMMAND step. Could be nicer for CMake devs, not a blocker though